io.github.dummerle.rare.json

```diff
@@ -0,0 +1,48 @@
+{
+    "app-id": "io.github.dummerle.rare",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "/app/bin/rare",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--filesystem=/usr/bin",
```

Comment:
nope. this will not be available.

`/usr/bin` is part of the runtime inside the sandbox